scripts/install-ops-agent.sh

```
@@ -0,0 +1,2 @@
+curl -sSO https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)